book-src/tour/custom-types.md

```diff
@@ -102,6 +102,18 @@ let Points(p) = score
 p // => 50
 ```
 
+## Named accessors
+
+If a custom type has only one varient and named fields they can be accessed
```

Comment:
variant

Reply:
thanks, are you able to write suggestions to PR's or is that only louis?

Reply:
```suggestion
If a custom type has only one variant and named fields they can be accessed
```